### ISSUE TITLE
Day Planner

### ISSUE THREAD

**Issue by kangkyu** [kangkyu/session-do#9]

**Comment by kangkyu** [comment]
There should be "one-time" task, not simply deleted, only check-marked after done. (why it should be all repetitive in the first place? The main thing should be one and only "timing the delay") This "Day Planner" will be a page listing tasks, also getting new task inputs, by input field or by picking existing tasks (one-time or repetitive)

**Comment by kangkyu** [comment]
If related 
+ set-when
+ done-when
+ still show after done with strikethrough
+ show logs on show page

I would prefer all these.

**Comment by kangkyu** [comment]
Today title on the main page
Strike-through with checkmarks
Tasks not done (daily or non-daily) and it goes to next day

Daily task: remove strikethrough on the next day
One-Time task: doesn't show on the next day (you delete manually after done, for now)
Non-daily / not-a-one-time task: stay to show on the next day
need a due-date for some tasks (should be done before a certain date)
(should be done AFTER a date for some too)
Done tasks: done_at or visited_at fields should be added
